src/services/addresses/getAddressParts.ts

```diff
@@ -0,0 +1,63 @@
+interface AddressParts {
```

Comment:
since this file is only used on the address search input let's move it to the folder there. Unless we start having shared hooks let's avoid spreading the logic.